basic_counter/lib/basic_counter_web/live/reaction_counter.ex

```diff
@@ -0,0 +1,77 @@
+defmodule CountersWeb.Live.ReactionCounter do
+  use CountersWeb, :live_component
+  @topic "reactions"
+
+  def render(assigns) do
+    ~H"""
+    <div class="my-4">
+      <h3 class="text-lg">
+        <span class="text-2xl"><%= @emoji %></span><%= render_slot(@inner_block) %>
+      </h3>
+      <div>
+        <button
+          class="bg-accent p-2 w-20 text-accent-foreground font-bold rounded-lg"
+          phx-click="inc"
+          phx-target={@myself}
+        >
+          🤩 <%= @count %>
+        </button>
+        Write took <%= @write_time |> us_to_ms %> ms, avg:
+        <%= @write_time_history |> get_avg_write_time |> us_to_ms %> ms
+      </div>
+    </div>
+    """
+  end
+
+  defp get_avg_write_time([]), do: 0
```

Comment:
I'd recommend going with `nil` and not displaying, probably makes more sense.

Reply:
Added a nil clause per you recommendation, and an empty UI state to match:
![image](https://user-images.githubusercontent.com/1357760/165100956-1b36aa3d-d913-46df-9ab7-6a9b6823c0d4.png)